packages/cli/src/commands/index.ts

```diff
@@ -217,12 +217,36 @@ export function registerCommands(program: Command) {
     .action(lazy(() => import('./create/create').then(m => m.default)));
```

Comment:
might be time to rename this folder and file already, since it reflects the current state of things

Reply:
Yep, why not!